matter/efr32/efr32mg24/BRD4187C/autogen/sl_spidrv_init.c

```diff
@@ -2,47 +2,49 @@
 #include "sl_spidrv_instances.h"
 #include "sl_assert.h"
 
-
-#ifdef RS911X_WIFI // EUSART is used in MG24 + RS9116 combination
+// EUSART is used in MG24 + RS9116 combination
+#ifdef RS911X_WIFI
+#if !(defined(EXP_BOARD) && EXP_BOARD)
 #include "sl_spidrv_eusart_exp_config.h"
 
 SPIDRV_HandleData_t sl_spidrv_eusart_exp_handle_data;
 SPIDRV_Handle_t sl_spidrv_eusart_exp_handle = &sl_spidrv_eusart_exp_handle_data;
 
 SPIDRV_Init_t sl_spidrv_eusart_init_exp = {
-  .port = SL_SPIDRV_EUSART_EXP_PERIPHERAL,
-  .portTx = SL_SPIDRV_EUSART_EXP_TX_PORT,
-  .portRx = SL_SPIDRV_EUSART_EXP_RX_PORT,
+  .port    = SL_SPIDRV_EUSART_EXP_PERIPHERAL,
+  .portTx  = SL_SPIDRV_EUSART_EXP_TX_PORT,
+  .portRx  = SL_SPIDRV_EUSART_EXP_RX_PORT,
   .portClk = SL_SPIDRV_EUSART_EXP_SCLK_PORT,
 #if defined(SL_SPIDRV_EUSART_EXP_CS_PORT)
   .portCs = SL_SPIDRV_EUSART_EXP_CS_PORT,
 #endif
-  .pinTx = SL_SPIDRV_EUSART_EXP_TX_PIN,
-  .pinRx = SL_SPIDRV_EUSART_EXP_RX_PIN,
+  .pinTx  = SL_SPIDRV_EUSART_EXP_TX_PIN,
+  .pinRx  = SL_SPIDRV_EUSART_EXP_RX_PIN,
   .pinClk = SL_SPIDRV_EUSART_EXP_SCLK_PIN,
 #if defined(SL_SPIDRV_EUSART_EXP_CS_PIN)
   .pinCs = SL_SPIDRV_EUSART_EXP_CS_PIN,
 #endif
-  .bitRate = SL_SPIDRV_EUSART_EXP_BITRATE,
-  .frameLength = SL_SPIDRV_EUSART_EXP_FRAME_LENGTH,
-  .dummyTxValue = 0,
-  .type = SL_SPIDRV_EUSART_EXP_TYPE,
-  .bitOrder = SL_SPIDRV_EUSART_EXP_BIT_ORDER,
-  .clockMode = SL_SPIDRV_EUSART_EXP_CLOCK_MODE,
-  .csControl = SL_SPIDRV_EUSART_EXP_CS_CONTROL,
+  .bitRate        = SL_SPIDRV_EUSART_EXP_BITRATE,
+  .frameLength    = SL_SPIDRV_EUSART_EXP_FRAME_LENGTH,
+  .dummyTxValue   = 0,
+  .type           = SL_SPIDRV_EUSART_EXP_TYPE,
+  .bitOrder       = SL_SPIDRV_EUSART_EXP_BIT_ORDER,
+  .clockMode      = SL_SPIDRV_EUSART_EXP_CLOCK_MODE,
+  .csControl      = SL_SPIDRV_EUSART_EXP_CS_CONTROL,
   .slaveStartMode = SL_SPIDRV_EUSART_EXP_SLAVE_START_MODE,
 };
 
-void sl_spidrv_init_instances(void) {
+void sl_spidrv_init_instances(void)
+{
 #if !defined(SL_SPIDRV_USART_EXP_CS_PIN)
   EFM_ASSERT(sl_spidrv_eusart_init_exp.csControl == spidrvCsControlAuto);
-#endif 
+#endif
   SPIDRV_Init(sl_spidrv_eusart_exp_handle, &sl_spidrv_eusart_init_exp);
 }
-
+#endif
 #endif

```

Comment:
can we add the comments for which #if this applies to

Reply:
added